auth/couchdbSessionAuthenticator.ts

```diff
@@ -19,6 +19,16 @@ import {
   SessionTokenManager,
   SessionTokenManagerOptions,
 } from './sessionTokenManager';
+import { Cookie, CookieJar, Store } from 'tough-cookie';
+
+interface SessionCookieJar extends CookieJar {
+  cloudantPatch: boolean;
+  store: Store;
+}
+
+class SessionCookie extends Cookie {
```

Comment:
WDYT about changing this also to `interface`?
```suggestion
interface SessionCookie extends Cookie {
```

Reply:
Yes! Nice catch! With that the compiled diff reduced to this:
```diff
diff --git a/couchdbSessionAuthenticator.js b/couchdbSessionAuthenticator.js
index 46a254f..48f08cf 100644
--- a/couchdbSessionAuthenticator.js
+++ b/couchdbSessionAuthenticator.js
@@ -1,6 +1,6 @@
 "use strict";
 /**
- * © Copyright IBM Corporation 2021. All Rights Reserved.
+ * © Copyright IBM Corporation 2021, 2022. All Rights Reserved.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -116,7 +116,7 @@ var CouchdbSessionAuthenticator = /** @class */ (function (_super) {
                     // the creation time by passing it as the now argument.
                     return originalExpiryTimeFn.call(newCookie, newCookie.cloudantPatchUpdateTime || now);
                 };
-                // Finally delegate back to the original update function or the fallback put (which is set by Cookie
+                // Finally, delegate back to the original update function or the fallback put (which is set by Cookie
                 // when an update function is not present on the store). Since we always set an update function for our
                 // patch we need to also provide that fallback.
                 if (originalUpdateCookieFn_1) {

```